deploy/helm/kube-cleanup-operator/templates/rbac.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if and .Values.rbac.create .Values.rbac.global }}
```

Comment:
1) you need to add similar double `if` to the current rbac files as well to check (rbac.create and not global) 
2) please split this file to have 1 object per file

Reply:
1. sure, I thought about this but didn't do it to have things untouched. Will do.
2. I totally support this perfectionism, update is coming!